app/services/user.js

```diff
@@ -15,6 +16,10 @@ class UserService extends CrudService {
       }
     }
 
+    if (user.companyId) {
```

Comment:
Why do you have this here? When a user is created it has nothing to do with a company.